apps/posterizarr/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM docker.io/library/alpine:3.21
+
+# Build arguments
+ARG TARGETARCH
+ARG VENDOR
+ARG VERSION=1.9.34
+
+# Set environment variables
+ENV UMASK="0002" \
+    TZ="Etc/UTC" \
+    DEBCONF_NONINTERACTIVE_SEEN="true" \
+    DEBIAN_FRONTEND="noninteractive" \
+    POWERSHELL_DISTRIBUTION_CHANNEL="PSDocker" \
+    VIRTUAL_ENV="/lsiopy"
+
+# Install APK dependencies
+RUN apk add --no-cache imagemagick imagemagick-jpeg imagemagick-heic libjpeg-turbo powershell tzdata
```

Comment:
Combine all the RUN directives into one, this makes things a lot easier to clean up to reduce the resulting image size as also explained in my other comment.